compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -1955,4 +2002,232 @@ private BLangUserDefinedType addUserDefinedType(DiagnosticPos pos,
         String[] pkgParts = sourcePkg.split("\\.|\\\\|\\/");
         return Arrays.asList(pkgParts);
     }
+
+    public void startOrderByClauseNode(DiagnosticPos pos, Set<Whitespace> ws) {
+        OrderByNode orderByNode = TreeBuilder.createOrderByNode();
+        ((BLangOrderBy) orderByNode).pos = pos;
+        ((BLangOrderBy) orderByNode).addWS(ws);
+        this.orderByClauseStack.push(orderByNode);
+    }
+
+    public void endOrderByClauseNode(DiagnosticPos currentPos, Set<Whitespace> ws) {
+        if (this.exprNodeListStack.empty()) {
+            throw new IllegalStateException("ExpressionList stack cannot be empty in processing an OrderBy clause");
```

Comment:
I think we don't need to throw exception here. Is there any chance we get to this state? I think these errors will be captured as grammer issues before getting to here. Check all the places

Reply:
removed all such occurrences.